### PR TITLE
Warnings clean up, dependency updates, and migration of settings.py

### PR DESCRIPTION
**Pull request by adamclmns**
Updated settings.py to remove warnings on startup about TEMPLATE settings.

Enabled Symposion schedule application. Admin site tested. Need template for Front-end site. 

Updated urls.py to use list of url() instead of patterns. 

Updated dependencies to avoid warnings on metaclass.

### CODE REVIEW

File: pyarkconf/settings.py [hunk 1]
```diff
@@ -5,9 +5,8 @@
 PACKAGE_ROOT = os.path.abspath(os.path.dirname(__file__))
 BASE_DIR = PACKAGE_ROOT
 
-DEBUG = True
-TEMPLATE_DEBUG = DEBUG
-EMAIL_DEBUG = DEBUG
+DEBUG = True #TODO: Change before deployment
```
Comment (calebrash) [inline pyarkconf/settings.py]: Added #17 to deal with these `TODO` comments.

File: pyarkconf/templates/schedule/schedule_edit.html [hunk 2]
```diff
@@ -20,7 +20,7 @@ <h1>Schedule Edit</h1>
         <h2>{{ timetable.day.date }}</h2>
         {% include "schedule/_edit_grid.html" %}
     {% endfor %}
-    <form id="schedule-builder" action="." method="post" enctype="multipart/form-data">{% csrf_token %}
+    <form id="schedule-builder" action="" method="post" enctype="multipart/form-data">{% csrf_token %}
```
Comment (calebrash) [inline pyarkconf/templates/schedule/schedule_edit.html]: Maybe `action="{{ request.path }}"`? I don't like trusting the browser to do the right thing.

File: requirements.txt [hunk 3]
```diff
@@ -25,4 +25,4 @@ psycopg2
 git+https://github.com/pinax/pinax-boxes@c69829f653d37bf090d1f62fb809824eb467c3bf#egg=pinax-boxes
 
 # symposion
-git+https://github.com/pinax/symposion.git#egg=symposion
+git+https://github.com/adamclmns/symposion.git#egg=symposion
```
Comment (bheklilr) [inline requirements.txt]: Do we need to review any changes coming from your versino of symposion?

File: pyarkconf/templates/symposion/schedule/_edit_grid.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<table class="calendar table table-bordered">
```
Comment (bheklilr) [inline pyarkconf/templates/symposion/schedule/_edit_grid.html]: Was this all freshly written code or did you have a reference implementation from somewhere?

File: requirements.txt (hunk 3 above)
Comment (adamclmns) [reply requirements.txt]: I haven't made any yet. Mostly seeing if it would work.

File: pyarkconf/templates/schedule/schedule_edit.html (hunk 2 above)
Comment (adamclmns) [reply pyarkconf/templates/schedule/schedule_edit.html]: I did not review these. They were straight copy/paste from what was already here. If we decide to change it here, we'll need to change it everywhere.

**Pull request merged**